pkgs/applications/misc/fuzzel/default.nix

```diff
@@ -13,29 +13,24 @@
 , tllist
 , fcft
 , enableCairo ? true
-, enablePNG ? true
-, enableSVG ? true
+, withPNGBackend ? "libpng"
+, withSVGBackend ? "librsvg"
```

Comment:
The meson_options upstream have been updated[1] to use a combo type, thus it no longer makes sense to restrict the value to boolean values. The nanorsvg library can now be used for the svg backend, a boolean value makes it impossible to distinguish between building with no svg support and building with the nanorsvg library. Adding additional options corresponding to each choice behavior is possible but I think it unnecessary complicates logic in the configuring phase, especially if dependencies form between options.

I agree an assertion is needed to emulate enum behavior rather than having the option as a freeform string.

[1] - https://codeberg.org/dnkl/fuzzel/commit/282ebce5b26dcd3ad4c29dbecb60b6f0fee5f943

Reply:
Usually these options are modeled after what is supported in nixpkgs and as long as there is only one possible value I personally wouldn't have changed it. I think if you want to more options that is fine but not required and I don't see the usecase for it right now. But I think the assert is needed here to prevent easy preventable errors.